packages/react/src/components/Button/Button.js

```diff
@@ -214,6 +215,9 @@ const Button = React.forwardRef(function Button(
     component = 'a';
     otherProps = anchorProps;
   }
+
+  const childrenWithText = <Text as="span">{children}</Text>;
```

Comment:
Ah, does Button potentially accept more than text children? This may be something where we would recommend that the consumer who wants the text translated either sets the direction themself or uses the `Text` component